doc/backup.md

```diff
@@ -5,6 +5,8 @@ To backup a LXD instance different strategies are available.
 ## Full backup
 This requires that the whole `/var/lib/lxd` folder will be backuped up.
 Additionally, it is necessary to backup all storage pools as well.
+The filesystem used for backups must support symlinks.
```

Comment:
Yeah, I was pointing this out to him because we've had an issue where someone did a `cp -R` to a FAT32 fs and lost all his symlinks.